### ISSUE TITLE
ДЗ в форке, вместо r-class репозитория

### ISSUE THREAD

**Issue by agricolamz** [zijane/2019_2020_ds4dh_hw3_purr_gutenbergr_tidytext_udpipe#2]
перенесите пожалуйста. Я не могу

* автоматически забирать задание
* пушить в этот репозиторий.

**Comment by zijane** [comment]
@agricolamz  да, поняла свою ошибку. Вот ссылка на нормальный  https://github.com/r-classes/ds4dh-hw3-zijane
